rules/aip0191/aip0191.go

```diff
@@ -45,3 +45,4 @@ func hasPackage(f *desc.FileDescriptor) bool {
 }
 
 var versionRegexp = regexp.MustCompile("^v[0-9]+(p[0-9]+)?((alpha|beta)[0-9]*)?$")
+var validCharacterRegexp = regexp.MustCompile("^[a-zA-Z 0-9\\_\\/]*$")
```

Comment:
There is a whitespace in between `A-Z` and `0-9`, is that necessary?

Reply:
Great catch! I also had a doubt about `A-Z`. Do we allow uppercase?

Reply:
Probably not, so lets remove it for now, and if it ends up being "OK" we can add it back (always easier to make it less restrictive later).